app/ashlar/migrations/0001_initial.py

```diff
@@ -35,7 +35,7 @@ class Migration(migrations.Migration):
                 ('occurred_to', models.DateTimeField()),
                 ('label', models.CharField(max_length=50)),
                 ('slug', models.CharField(max_length=50)),
-                ('geom', django.contrib.gis.db.models.fields.PointField(srid=b'3857')),
+                ('geom', django.contrib.gis.db.models.fields.PointField(srid=3857)),
```

Comment:
Since I had to make this change in the actual migration file, you'll probably want to roll back your migrations to init, then update to this branch, then migrate forwards again.